en/ios/user-interface.mdown

```diff
@@ -484,8 +484,56 @@ The easiest way to understand this class is with an example. This subclass of `P
 @end
 ```
 ```swift
-// no Swift example
-// Want to contribute to this doc? https://github.com/ParsePlatform/Docs
+class SimpleTableViewController : PFQueryTableViewController {
+    
+    override init(style: UITableViewStyle, className: String?) {
+        super.init(style: style, className: className)
+        self.parseClassName = "Todo"
+        self.pullToRefreshEnabled = true
+        self.paginationEnabled = true
+        self.objectsPerPage = 25
+    }
+
+    required init!(coder aDecoder: NSCoder!) {
```

Comment:
I believe the method signature for this one is `public init?(coder aDecoder: NSCoder)` aka it could return `nil` as well as coder will never be `nil`.